cross_compile.md

```diff
@@ -32,9 +32,7 @@ yum install mingw64-openssl-static mingw64-zlib-static mingw64-winpthreads-stati
             .define("CMAKE_SYSTEM_PROCESSOR", get_env("CARGO_CFG_TARGET_ARCH").unwrap())
             .define("CMAKE_CROSSCOMPILING", "true")
             .build_target(library)
```

Comment:
Only keep the change in this file and reset all others by `git checkout master -- src` should fix CI.

Reply:
It was a strange circular bug that wasn't resolved by that for some reason... Didn't like indentation in the doclist item of 2, nor 3, nor 0. Anyway, it's resolved now 🤞 Thx again